### ISSUE TITLE
V2 : Missing JAR artifact in JCenter

### ISSUE THREAD

**Issue by ncornette** [yigit/android-priority-jobqueue#301]
Please publish JAR artifact in JCenter and not only AAR. 

As jobQueue library doesn't provide any resource or asset it should definitely be a JAR. 

JAR provides more flexibility when including Jobqueue into a library as other related libraries are JARs : Okhttp, Retrofit, Eventbus, Rxjava, etc...

**Comment by ncornette** [comment]
Created PR #302 it works but some arguments have been duplicated, please review !

**Comment by yigit** [comment]
We have permission definitions in the manifest:

https://github.com/yigit/android-priority-jobqueue/blob/master/jobqueue/src/main/AndroidManifest.xml

**Comment by yigit** [comment]
I have another suggestion. Instead of providing a jar alternative, lets create a new artifact that has a different name. Something like "android-priority-jobqueue-jar". This way, people won't depend on the jar by mistake (because aar should be to avoid permissions problems)

**Comment by ncornette** [comment]
I agree with your suggestion, the goal being to make the jar accessible!
